doc/api/debugger.md

```diff
@@ -184,12 +184,14 @@ localhost:5858
 
 ## V8 Inspector Integration for Node.js
 
+__NOTE: This is an experimental feature.__
+
 V8 Inspector integration allows attaching Chrome DevTools to Node.js
 instances for debugging and profiling.
 
-V8 Inspector can be enabled by passing `--inspect` flag when starting Node.js
-application. It is also possible to supply a custom port with that flag, e.g.
-`--inspect=9222` will expect DevTools connection on the port 9222.
+V8 Inspector can be enabled by passing the `--inspect` flag when starting a
+Node.js application. It is also possible to supply a custom port with that flag,
+e.g. `--inspect=9222` will expect DevTools connection on the port 9222.
```

Comment:
This sentence reads a little awkward.  How about "will accept DevTools connections on port 9222."

Or "accept a DevTools connection" if it only accepts one (which it does, I believe.)